docs/user/spark/jupyter.rst

```diff
@@ -47,7 +52,8 @@ or
 Installing the Toree Kernel
 ---------------------------
 
-Toree version 0.2.0-dev1 supports Spark 2.0 and above, and should be built from source as described via its `README file`_. Check out the project from GitHub:
+Toree version 0.2.0-dev1 supports Spark 2.0 and above, and should be built from source as described via
```

Comment:
@aheyne @tkunicki We might should review this bit about using Toree...

Reply:
IMO we should just ditch Toree and/or strongly recommend Zeppelin for anything Scala and Jupyter for only Python uses.